src/edu/csus/ecs/pc2/core/imports/clics/FieldCompareRecord.java

```diff
@@ -0,0 +1,146 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.report.ComparisonState;
+import edu.csus.ecs.pc2.core.report.FileComparisonUtilities;
+
+/**
+ * Information about a comparison of two fields.
+ * 
+ * @author Douglas A. Lane, PC^2 team pc2@ecs.csus.edu
+ */
+public class FieldCompareRecord {
+
+    @JsonProperty
+    private ComparisonState state = ComparisonState.UNDEFINED;
+
+    @JsonProperty
+    private String key = null;
+
+    @JsonProperty
+    private String fieldName;
+
+    @JsonProperty
+    private String valueOne;
+
+    @JsonProperty
+    private String valueTwo;
+
+    @JsonProperty
+    private String details;
+
+    public FieldCompareRecord(String fieldName, String valueOne, String valueTwo, String details) {
+        this(fieldName, valueOne, valueTwo, details, null);
+    }
+    
+    public FieldCompareRecord(String fieldName, String valueOne, String valueTwo, String details, String key) {
+        super();
```

Comment:
This statement is superfluous (no pun intended) -- every Java constructor implicitly calls `super()` (unless it calls another constructor as its first statement).